itou/asp/migrations/0006_alter_commune_managers_commune_ignore.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 5.1.8 on 2025-05-06 15:21
+
+from django.db import migrations, models
+
+import itou.asp.models
+
+
+def forward(apps, editor):
+    Commune = apps.get_model("asp", "Commune")
+    # Exotic code existing only for the ASP.
+    # https://www.insee.fr/fr/metadonnees/geographie/commune/97411-saint-denis
+    obsolete_st_denis = Commune.objects.filter(code="97811")  # Saint-Denis
+    obsolete_st_denis.update(ignore=True)
+    active_st_denis = Commune.objects.filter(code="97411").current().last()
+    JobSeekerProfile = apps.get_model("users", "JobSeekerProfile")
+    JobSeekerProfile.objects.filter(birth_place_id__in=obsolete_st_denis.values_list("pk")).update(
+        birth_place_id=active_st_denis.pk
+    )
```

Comment:
Vu qu'on est dans une transaction ça ne devrais rien changer _in fine_ mais ça reste mieux de migrer les données encore utilisées avant de les cacher :).
```suggestion
    active_st_denis = Commune.objects.filter(code="97411").current().last()
    JobSeekerProfile = apps.get_model("users", "JobSeekerProfile")
    
    JobSeekerProfile.objects.filter(birth_place_id__in=obsolete_st_denis.values_list("pk")).update(
        birth_place_id=active_st_denis.pk
    )
    obsolete_st_denis.update(ignore=True)
```